torch_np/tests/numpy_tests/lib/test_arraysetops.py

```diff
@@ -857,6 +856,15 @@ def _run_axis_tests(self, dtype):
         result = np.array([[0, 0, 1], [0, 1, 0], [0, 0, 1], [0, 1, 0]])
         assert_array_equal(unique(data, axis=1), result.astype(dtype), msg)
 
+        pytest.xfail("torch has different unique ordering behaviour")
+        # e.g.
+        #
+        #     >>> x = np.array([[[1, 1], [0, 1]], [[1, 0], [0, 0]]])
+        #     >>> np.unique(x, axis=2)
+        #    [[1, 1], [0, 1]], [[1, 0], [0, 0]]
+        #     >>> torch.unique(torch.as_tensor(x), dim=2)
+        #    [[1, 1], [1, 0]], [[0, 1], [0, 0]]
+        #
```

Comment:
Is this something we need to fix in our unique?

The Notes section in https://numpy.org/doc/stable/reference/generated/numpy.unique.html is probably relevant, if somewhat obscure.

WDYT @lezcano ?

Reply:
(`unique` against axis gave me a headache so I had left this xfail'd and documented incase you/Mario had an immediate answer :sweat_smile:)

Reply:
Uf. This looks like a bit of a pain to implement. I'd say we don't touch it and leave it xfailed.